docs/src/index.rst

```diff
@@ -26,6 +26,13 @@ Developers can use one of the supported programming languages to define reusable
 cloud components called :doc:`constructs`, which are composed together into
 :doc:`stacks` and :doc:`apps`.
 
+.. note::
+    Unless otherwise indicated,
+    the code examples in this guide are in TypeScript.
+
+    To aid you in porting a TypeScript example to a supported programming language,
+    take a look at the example code for your language in the reference section for the service.
```

Comment:
"the reference section of the service"?, maybe just "the reference section" (with a link)

Reply:
Yep, make it easier for the reader.